nvflare/private/fed/app/simulator/simulator.py

```diff
@@ -28,6 +28,7 @@ def define_simulator_parser(simulator_parser):
     simulator_parser.add_argument("-c", "--clients", type=str, help="client names list")
     simulator_parser.add_argument("-t", "--threads", type=int, help="number of parallel running clients")
     simulator_parser.add_argument("-gpu", "--gpu", type=str, help="list of GPU Device Ids, comma separated")
+    simulator_parser.add_argument("-m", "--max_clients", type=int, default=100, help="max number of clients")
```

Comment:
We might want to change this "threads" argument name?
Suggest to `n_parallel_clients`

Reply:
Simulator already have been officially released. Should not change this to break the user facing argument name.